README.md

```diff
@@ -281,11 +268,7 @@ Else you can start with [CONTRIBUTING.md](CONTRIBUTING.md).
 * Document `Pricing` features
 * Add formatCurrency component
 * Extract babeljs compilation process from extract_messages script (CRA compatible)
-* Add a script launcher and remove usage of `node_modules/.bin/bla-thing` from docs (cf. CRA)
 * Improve doc on how to load translations (at least give an example)
 * Link translator options to related libs
-* Remove required pybabel.cfg
 * Allow user to init a logger for missing translations and use `warning` as fallback
 * Fix or change build process in `bin/merge_catalogs` which looks for legacy gandi's catalogs
-* Replace pybabel by a npm script (eg. https://www.npmjs.com/package/babel-gettext-extractor)
-* Fix dependency path for scripts called in extract_messages (e.g. po2json)
```

Comment:
@iamdey : does it still extract translation strings from lib dependencies that also use react-translate?